### ISSUE TITLE
Error when adding ssl cert to elb on AWS

### ISSUE THREAD

**Issue by aldarund** [hashicorp/terraform#4614]
I have added following into my tf conf.

```
resource "aws_iam_server_certificate" "rent_cert" {
  name = "some_test_cert"
  certificate_body = "${file("ssl-cert.pem")}"
  private_key = "${file("ssl-key.pem")}"
}
```
and into elb conf
```
  listener {
    instance_port = 9999
    instance_protocol = "HTTP"
    lb_port = 443
    lb_protocol = "https"
    ssl_certificate_id = "${aws_iam_server_certificate.rent_cert.arn}"
  }
```
After applying plan i get following error:

Error applying plan:

1 error(s) occurred:

* aws_elb.rentapp_service: Failure adding new or updated ELB listeners: CertificateNotFound: Server Certificate not found for the key: arn:aws:iam::133662108267:server-certificate/some_test_cert
	status code: 400, request id: 1c101d7d-b7e5-11e5-b996-efd4aa145c43

But in logs there was:
```
aws_iam_server_certificate.rent_cert: Creating...
  arn:              "" => "<computed>"
  certificate_body: "" => "c35c92ef60c0e74ebdab7a9fab020f41c3d4bef7"
  name:             "" => "some_test_cert"
  path:             "" => "/"
  private_key:      "" => "47d42cf5a4df2c7cc198f5d28d713c8f85e161c1"
.....
aws_iam_server_certificate.rent_cert: Creation complete
```
After applying second time without changing anything the error is gone and its applied succesfully

**Comment by saliceti** [comment]
Same issue here. With Terraform 0.6.10

**Comment by catsby** [comment]
Hi friends – this will be patched with #5178, where I've added a retry for this case

**Comment by stack72** [comment]
Hi @aldarund, this has been fixed now that #5178 has been merged. It will be available in Terraform 0.6.12 in 1 or 2 days
